package.json

```diff
@@ -29,50 +29,39 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch:lib": "watch 'npm run lint && npm test' ./lib"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "body-parser": "~1.11.0",
+    "async": "2.6.1",
+    "body-parser": "~1.18.3",
     "command-shell-lib": "1.0.0",
-    "express": "~4.11.2",
-    "jison": "0.4.17",
-    "logops": "1.0.0",
-    "moment": "~2.19.4",
+    "express": "~4.16.4",
+    "jison": "0.4.18",
+    "jshint": "^2.9.6",
```

Comment:
jshint should be moved to devDependencies? As far as I understand, it is not needed for production.

Reply:
Fixed with 98f499d69a0c4dd6fba625af4b763ba6e17fe18d